docs/docfx.json

```diff
@@ -0,0 +1,66 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "files": [
+            "proknow-sdk/proknow-sdk.csproj"
+          ],
+          "cwd": ".."
+        }
+      ],
+      "dest": "api",
+      "disableGitFeatures": false,
+      "disableDefaultFilter": false,
+      "force": true
+    }
+  ],
+  "build": {
+    "content": [
+      {
+        "files": [
+          "api/**.yml",
+          "api/index.md"
+        ]
+      },
+      {
+        "files": [
+          "articles/**.md",
+          "articles/**/toc.yml",
+          "toc.yml",
+          "*.md"
+        ]
+      }
+    ],
+    "resource": [
+      {
+        "files": [
+          "images/**"
+        ]
+      }
+    ],
+    "overwrite": [
+      {
+        "files": [
+          "apidoc/**.md"
+        ],
+        "exclude": [
+          "obj/**",
+          "_site/**"
+        ]
+      }
+    ],
+    "dest": "_site",
+    "globalMetadataFiles": [],
+    "fileMetadataFiles": [],
+    "template": [
+      "default"
+    ],
+    "postProcessors": [],
+    "markdownEngineName": "markdig",
+    "noLangKeyword": false,
+    "keepFileLink": false,
+    "cleanupCacheHistory": false,
+    "disableGitFeatures": false
+  }
+}
```

Comment:
I added .editorconfig files.  I couldn't add one at the solution level because, in the solution, files at the root are actually in a solution folder called "Solution Items."  I can't change this.  When the .editorconfig file is there, it doesn't seem to work because everything else is in parallel folders.  So I put one in the docs folder and one in the proknow-sdk folder.  Note that I couldn't put one in the proknow-sdk-test folder because it squawked and said I was overriding configuration from an imported library.  Sigh...  The configurations in .editorconfig are only applied when the file is formatted or saved, so I still need to manually fix (save) each one that is currently missing the newline.

Reply:
I don't know what the issue with proknow-sdk-test was.  Probably a user error or undocumented feature error (Visual Studio likes to keep the solution folder hierarchy different than the one on disk to make things more interesting.  I tried again and was able to put a .editorconfig file in this folder as well.